commands_function.py

```diff
@@ -0,0 +1,59 @@
+import os
+from abc import ABCMeta, abstractstaticmethod
+import json
+from keys import keys
+
+class Icommand(metaclass=ABCMeta):
+    @abstractstaticmethod
+    def excute():
+        """the interface Icommand"""
+
+class commandfactory:
+    @staticmethod
+    def build_command(args):
+        command_type=args.command
```

Comment:
add space before and after the '=' operator

Reply:
ok we will

Reply:
You are create command not build it. Because there is two design battern regarding to object creation :- 
1- Factory design pattern 
2- Builder design pattern. 

Here, you are creating the object, not building it

Reply:
@mohamedabdelsalam sorry can you explain more.